command/service/ls.go

```diff
@@ -0,0 +1,91 @@
+package service
+
+import (
+	"fmt"
+	"sort"
+	"strings"
+
+	"github.com/craftcms/nitro/pkg/containerlabels"
+	"github.com/docker/docker/api/types"
+	"github.com/docker/docker/api/types/filters"
+	"github.com/docker/docker/client"
+	"github.com/rodaine/table"
+	"github.com/spf13/cobra"
+
+	"github.com/craftcms/nitro/pkg/terminal"
+)
+
+func lsCommand(home string, docker client.CommonAPIClient, output terminal.Outputer) *cobra.Command {
+	cmd := &cobra.Command{
+		Use:     "ls",
+		Aliases: []string{"l"},
+		Short:   "Lists details for Nitro’s services.",
+		Example: `  # view information about your nitro environment
+  nitro service ls`,
+		RunE: func(cmd *cobra.Command, args []string) error {
+			ctx := cmd.Context()
+
+			// add filters to show only the environment and database containers
+			filter := filters.NewArgs()
+			filter.Add("label", containerlabels.Nitro)
+
+			// get a list of all the databases
+			containers, err := docker.ContainerList(ctx, types.ContainerListOptions{All: true, Filters: filter})
+			if err != nil {
+				return err
+			}
+
+			// sort containers by the name
+			sort.SliceStable(containers, func(i, j int) bool {
+				return containers[i].Names[0] < containers[j].Names[0]
+			})
+
+			// define the table headers
+			tbl := table.New("Hostname", "Type", "External Ports", "Internal Ports", "Status").WithWriter(cmd.OutOrStdout()).WithPadding(2)
+
+			for _, c := range containers {
+				status := "running"
+				if c.State == "exited" {
+					status = "stopped"
+				}
+
+				// show only the service containers
+				if c.Labels[containerlabels.Type] != "dynamodb" && c.Labels[containerlabels.Type] != "mailhog" && c.Labels[containerlabels.Type] != "redis" && c.Labels[containerlabels.Type] != "blackfire" {
```

Comment:
Similar to https://github.com/craftcms/nitro/pull/433/files#r754689737, should we have a more global const we check against for every time we need to filter these? Seems like it's duplicated in several places https://github.com/craftcms/nitro/pull/433/files#diff-6932c534e4813e74151d19d052361f6d7d2e90e454a485c897cac1fb1dea49c7R33

Reply:
Resolved in https://github.com/craftcms/nitro/pull/433/commits/8b27be1e38e017536bbeb2dbdea81a794a9efdd2.